widgetssdk/src/main/java/com/glia/widgets/chat/Intention.kt

```diff
@@ -0,0 +1,24 @@
+package com.glia.widgets.chat
+
+internal enum class Intention {
+    RESTORE_CHAT,
+    SC_DIALOG_START_AUDIO,
+    SC_DIALOG_START_VIDEO,
```

Comment:
Just an idea. `SC_DIALOG_START_AUDIO` and `SC_DIALOG_START_VIDEO` sound more like the place from where the call originated. To sound more like an intention, maybe these names might be better: `CHAT_FROM_SC_AUDIO` and `CHAT_FROM_SC_VIDEO`?

Reply:
I think it sounds exactly like I wanted.
`SC_DIALOG_START_AUDIO` intention is - to open SC chat with the `leave sc` dialog and then, based on user input, stay in SC chat or start the audio call.

Maybe it makes sense to add a description for each intention?

Reply:
Ah, okay, agree. Yes, a description is definitely needed here.

Reply:
added